vmc/resource_vmc_sddc.go

```diff
@@ -398,7 +398,10 @@ func resourceSddcUpdate(d *schema.ResourceData, m interface{}) error {
 			action = "remove"
 			diffNum = oldNum - newNum
 		}
-
+		if d.Get("deployment_type").(string) == MultiAvailabilityZone && diffNum%2 != 0 {
+			log.Printf("Check for the diffnum %d", diffNum%2)
+			return fmt.Errorf("for multiAZ SDDC hosts must be added in pairs across availability zones")
```

Comment:
for more clarity, suggest re-wording error to - "for multiAZ deployment type, SDDC hosts must be added in pairs across availability zones"

Reply:
Thanks for reviewing the PR. I will make the changes

Reply:
Done